stable/aws-for-fluent-bit/templates/service.yaml

```diff
@@ -10,12 +8,10 @@ metadata:
 spec:
   ports:
   - name: monitor-agent
-    port: {{ .Values.serviceMonitor.service.port }}
+    port: {{ .Values.k8sService.port }}
```

Comment:
AFAICT, port and type for k8sService is not on FLB but is on the serviceMonitor, so to me it makes sense to keep it under the `serviceMonitor` config namespace?